### ISSUE TITLE
7 day Ad history shows the history of only the current day

### ISSUE THREAD

**Issue by robin-thomas** [brave/brave-browser#8926]
## Description 
7 day Ad history shows the history of only the current day. It gets reset every single day, so I can't see the history of more than 24 hours.

## Steps to Reproduce

Go to chrome://rewards/

## Actual result:

7 day Ad history shows the history of only the current day. It gets reset every single day, so I can't see the history of more than 24 hours.

## Expected result:
7 day Ad History page shows history of 7 days

## Reproduces how often: 

Every time

## Brave version (brave://version)

```sh
Brave | 1.5.115 Chromium: 80.0.3987.149 (Official Build) (64-bit)
Revision | 5f4eb224680e5d7dca88504586e9fd951840cac6-refs/branch-heads/3987_137@{#16}
OS | Linux
```

**Comment by NejcZdovc** [comment]
we have an issue here for supporting dark theme: https://github.com/brave/brave-browser/issues/4229. Will change this issue so that it's only about 7 day history

**Comment by robin-thomas** [comment]
@NejcZdovc @kjozwiak This issue has been resolved in the latest version: `Version 1.7.92 Chromium: 80.0.3987.163 (Official Build) (64-bit)`.

Hence, closing this.
